algebra/src/main/java/org/almostrealism/collect/computations/EpsilonConstantComputation.java

```diff
@@ -27,26 +27,160 @@
 
 import java.util.List;
 
+/**
+ * An {@link EpsilonConstantComputation} represents a computation that produces a 
+ * {@link PackedCollection} where every element contains the machine epsilon value.
+ * Machine epsilon is the smallest representable increment between two floating-point 
+ * numbers, providing a precision-aware constant for numerical computations.
+ * 
+ * <p>This class extends {@link SingleConstantComputation} but overrides the expression
+ * generation to use the actual machine epsilon value instead of a literal constant.
+ * While the constructor passes 0.0 to the parent class, the actual computation uses
+ * the precision-specific epsilon value obtained from {@link io.almostrealism.expression.Epsilon}.</p>
+ * 
+ * <h3>Primary Use Case:</h3>
+ * <p>EpsilonConstantComputation is primarily used for floating-point comparisons where
+ * numerical precision matters. It provides collections filled with epsilon values that
+ * can be used as tolerance thresholds in equality operations and other precision-sensitive
+ * computations.</p>
+ * 
+ * <h3>Usage Examples:</h3>
+ * <pre>{@code
+ * // Create a 3x3 matrix filled with machine epsilon values
+ * TraversalPolicy shape = new TraversalPolicy(3, 3);
+ * EpsilonConstantComputation<PackedCollection> epsilon = 
+ *     new EpsilonConstantComputation<>(shape);
+ * 
+ * // Use in floating-point equality comparison (typical usage)
+ * CollectionProducer<PackedCollection> a = ...; // some computation
+ * CollectionProducer<PackedCollection> b = ...; // another computation
+ * CollectionProducer<PackedCollection> areEqual = 
+ *     equals(a, b, new EpsilonConstantComputation<>(a.getShape()), 
```

Comment:
This is not how the equals method of CollectionFeatures behaves. Just remove this discussion entirely; we can come back to it when documenting the comparison computations.